examples/games/trex/lib/random_extension.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:math';
 
-Random random = Random();
+final Random random = Random();
```

Comment:
```suggestion
final random = Random();
```

Reply:
Hopefully there is a DCM rule for that too waiting for me to enable
![image](https://github.com/flame-engine/flame/assets/882703/c637d11f-e755-433a-abc0-512b0c033f77)